File: ReactNative/src/modules/todo/services/delete-todo.service.ts
import { TokenAxios } from '../../common/utils/token-axios';
import { TODO_URL } from '../../common/utils/url';

export async function requestDeleteTodo(idx: number): Promise<void> {
  const AXIOS = await TokenAxios();
  await AXIOS.delete(`${TODO_URL}/${idx}`);
}

File: flutter_todo/lib/src/modules/todo/services/delete_todo_service.dart
import 'package:flutter_todo/src/modules/common/utils/token_dio.dart';
import 'package:flutter_todo/src/modules/common/utils/url.dart';

Future<void> requestDeleteTodo(int idx) async => (await tokenDio())
    .delete('$todoUrl/$idx')
    .catchError((error) => throw error);
